handlers.go

```diff
@@ -44,10 +46,11 @@ func GetMux() *mux.Router {
 	r.HandleFunc(`/user-agent`, UserAgentHandler).Methods(http.MethodGet, http.MethodHead)
 	r.HandleFunc(`/headers`, HeadersHandler).Methods(http.MethodGet, http.MethodHead)
 	r.HandleFunc(`/get`, GetHandler).Methods(http.MethodGet, http.MethodHead)
+	r.HandleFunc(`/post`, PostHandler).Methods(http.MethodPost)
 	r.HandleFunc(`/redirect/{n:[\d]+}`, RedirectHandler).Methods(http.MethodGet, http.MethodHead)
 	r.HandleFunc(`/absolute-redirect/{n:[\d]+}`, AbsoluteRedirectHandler).Methods(http.MethodGet, http.MethodHead)
 	r.HandleFunc(`/redirect-to`, RedirectToHandler).Methods(http.MethodGet, http.MethodHead).Queries("url", "{url:.+}")
-	r.HandleFunc(`/status/{code:[\d]+}`, StatusHandler).Methods(http.MethodGet, http.MethodHead)
+	r.HandleFunc(`/status/{code:[\d]+}`, StatusHandler)
```

Comment:
there are more options like TRACE OPTIONS DELETE CONNECT. I think it'd be good to match httpbin here and just whitelist what's allowed rather than allowing all.

Reply:
httpbin.org supports the following methods: `OPTIONS, PUT, GET, DELETE, HEAD, POST, TRACE, PATCH`. 

I believe we should the same, except if it goes against the HTTP protocol. However I couldn't not find any explicit restriction https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html)

Is this ok with you ?

Reply:
we can keep it as is.